src/frontend/youtube_clone/src/app/main-page/main-page.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
We don't need unit tests now, so you can disable that generation in Angular schema and delete all `.spec` files

Reply:
done :)